scripts/mockdata/scenarios_e1.json

```diff
@@ -97,6 +97,28 @@
           }
```

Comment:
I think we're missing a few slides for the scene intro here 
![image](https://user-images.githubusercontent.com/35475783/139981380-9850774b-b42d-4ab6-b1a0-6dc4813cc6f1.png)

Reply:
Added 👌